src/ax/saga/persistence/eventsourcing/persister.go

```diff
@@ -40,6 +40,8 @@ func (p *Persister) BeginUnitOfWork(
 		pk string
 	)
 
+	uow := &unitOfWork{}
```

Comment:
Now that `uow` is declared early, there's no need to have the `i` and `pk` variables. I think it would be clearer to set the fields of `unitOfWork` when it is declared (for those fields that can be).